src/scripts/bttvIntegration.js

```diff
@@ -0,0 +1,105 @@
+function replaceKeywordWithEmoteImageString(textContent, keyword, imageSource, id){ 
+    if (!textContent) { 
+        throw "Text content is not provided!" 
+    } else if (typeof textContent !== "string"){ 
+        throw "Text content is not a string!"
+    }
+    const otherArgumentsAreValid = areValidInputs(keyword, imageSource, id); 
+    if (!otherArgumentsAreValid) {return textContent }
+
+    const imageHtmlString = `<img alt="${keyword}" data-twitch-emote="${keyword}" data-twitch-emote-id="${id}" src="${imageSource}"></img>`;
```

Comment:
It could be neat to add a bit of an API detailing the source of the emotes - perhaps `data-twitch-emote-source="bttv"` or something like that? I'm not sure if that'd be at all useful except maybe for debugging. Thoughts?

Reply:
You know that's a great idea actually! Not just debugging, but that'd be an easy way to allow streamers to config that with a toggle or something similar. Not necessarily all of them would want that now that I think about it

Reply:
I was thinking about making BTTV emotes a configurable thing so folks could either opt in or opt out! But I'll get that once I add configurations (prob later this week)